new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomain.java

```diff
@@ -0,0 +1,1284 @@
+   *
+   * @param domain oracle.weblogic.domain.Domain object
+   */
+  @ParameterizedTest
+  @DisplayName("scale cluster using WLDF policy for three different type of domains")
+  @MethodSource("domainProvider")
+  public void testParamsScaleClustersWithWLDF(Domain domain) {
+    assertDomainNotNull(domain);
+
+    // Verify scale cluster of the domain with WLDF policy
+    logger.info("testScaleClustersWithWLDF with domain {0}", domain.getMetadata().getName());
+    testScaleClustersWithWLDF(domain);
+  }
+
+  /**
+   * Verify admin console login using admin node port.
+   *
+   * @param domain oracle.weblogic.domain.Domain object
+   */
+  @ParameterizedTest
+  @DisplayName("Test admin console login using admin node port")
+  @MethodSource("domainProvider")
+  public void testAdminConsoleLoginUsingAdminNodePort(Domain domain) {
+    assertDomainNotNull(domain);
+    String domainUid = domain.getSpec().getDomainUid();
+    String domainNamespace = domain.getMetadata().getNamespace();
+    String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+
+    logger.info("Getting node port for default channel");
+    int serviceNodePort = assertDoesNotThrow(() -> getServiceNodePort(
+        domainNamespace, adminServerPodName + "-external", "default"),
+        "Getting admin server node port failed");
+
+    logger.info("Validating WebLogic admin server access by login to console");
+    boolean loginSuccessful = assertDoesNotThrow(() ->
+        adminNodePortAccessible(serviceNodePort, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT),
+        "Access to admin server node port failed");
+    assertTrue(loginSuccessful, "Console login validation failed");
+  }
+
+  /**
+   * Verify admin console login using ingress controller.
+   *
+   * @param domain oracle.weblogic.domain.Domain object
+   */
+  @ParameterizedTest
+  @DisplayName("Test admin console login using ingress controller")
+  @MethodSource("domainProvider")
+  public void testAdminConsoleLoginUsingIngressController(Domain domain) {
+    logger.info("Validating WebLogic admin server access using ingress controller");
+
+    assertDomainNotNull(domain);
+    String domainUid = domain.getSpec().getDomainUid();
+    String domainNamespace = domain.getMetadata().getNamespace();
+
+    String curlCmd = "curl --silent --show-error --noproxy '*' -H 'host: "
+        + domainUid + "." + domainNamespace + ".adminserver.test"
+        + "' http://" + K8S_NODEPORT_HOST + ":" + nodeportshttp
+        + "/console/login/LoginForm.jsp --write-out %{http_code} -o /dev/null";
+
+    logger.info("Executing curl command {0}", curlCmd);
+    assertTrue(callWebAppAndWaitTillReady(curlCmd, 60));
+    logger.info("WebLogic console on domain1 is accessible");
+  }
+
+  /**
+   * Verify liveness probe by killing managed server process 3 times to kick pod container auto-restart.
+   */
+  @ParameterizedTest
+  @DisplayName("Test liveness probe of pod")
+  @MethodSource("domainProvider")
```

Comment:
can you test this for only one type of domain, no need to test with all three models, just adds up run time.

Reply:
fixed